spec/factories.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+  sequence :lat do |n|
+    "#{n}.0".to_f
+  end
+
+  sequence :lon do |n|
+    "#{n}.0".to_f
```

Comment:
Instead of a sequence maybe could do something like:

`(rand * 179).round(3) * [-1,1].sample`

Or 90 for lats?